tests/test_eval_detection.py

```diff
@@ -34,41 +34,62 @@ def metrics(evaluator: DetectionEvaluator) -> DataFrame:
 
 def test_center_similarity() -> None:
     """Test that the Center similarity function works."""
-    olr1 = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([0, 0, 0]), 5.0, 5.0, 5.0, 0)])
-    olr2 = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([3, 4, 0]), 5.0, 5.0, 5.0, 0)])
-    assert compute_affinity_matrix(olr1, olr2, SimFnType.CENTER) == -5
+    dts = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([0, 0, 0]), 5.0, 5.0, 5.0, 0)])
+    gts = np.array([ObjectLabelRecord(np.array([0, 0, 0, 0]), np.array([3, 4, 0]), 5.0, 5.0, 5.0, 0)])
```

Comment:
compute_affinity_matrix should accept two lists, not two np array of objects:
https://github.com/argoai/argoverse-api/pull/159/commits/89dba090e158d8e68b3a8073c75d44b62282d1e5#diff-b1383b4f1990fd677aa53ec6e4170e2eR105